Source/Shaders/PostProcessFilters/AmbientOcclusion.glsl

```diff
@@ -0,0 +1,11 @@
+uniform sampler2D colorTexture;
+uniform sampler2D ambientOcclusionTexture;
```

Comment:
Although I used the term "filter" 5 or 6 years ago, I think "stage" is better.  Should the `PostProcessFilters` folder be renamed to `PostProcessStages`.

Also if a stages has 2-3+ .glsl files, I think it is fair game to make its own subdirectory, e.g., `PostProcessStages/AmbientOcclusion/Modulate.glsl`.